src/common/components/new-tab-link-with-tooltip.tsx

```diff
@@ -23,7 +26,10 @@ export const NewTabLinkWithTooltip = NamedFC<NewTabLinkWithTooltipProps>(
         };
         return (
             <TooltipHost content={tooltipContent} styles={hostStyles} calloutProps={calloutProps}>
-                <NewTabLink className={styles.insightsLink} {...linkProps} />
+                <NewTabLink
+                    className={typeof className === 'undefined' ? styles.insightsLink : className}
```

Comment:
why we need typeOf? we are checking className only rite?

Reply:
Yes, we have removed the typeOf as it is not required and updated the code.